### PR TITLE
UIREQ-372: Display effective call number string on the request queue …

### PR DESCRIPTION
**Pull request by SergiyVSergiyenko**
## Purpose
Display effective call number string on the dedicated request queue page. [Story](https://issues.folio.org/browse/UIREQ-372).

### Screenshots
**Item data**
Item_data

**Call number string**
Call_number_string

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -160,6 +160,7 @@
   },
   "dependencies": {
     "@folio/react-intl-safe-html": "^1.0.2",
+    "@folio/stripes-util": "^2.0.0",
```
Comment (zburke) [inline package.json]: Please import this via `@folio/stripes/util` instead.

Comment (zburke) [reply package.json]: i.e. we don't need a direct dependency because we can rely on the `@folio/stripes` peerDependency.

Comment (SergiyVSergiyenko) [reply package.json]: @zburke, I don't know why, but when importing via '@folio/stripes/util' the 'copyNumber' value doesn't appear in the 'Effective call number string' field.

Comment (zburke) [reply package.json]: What the ...? Makes me wonder if the correct version of stripes-util is being wrapped into stripes.

**Pull request closed without merging**